client/components/LearnerViewDirective.js

```diff
@@ -192,26 +205,52 @@ tie.directive('learnerView', [function() {
       </style>
     `,
     controller: [
-      '$scope', 'SolutionHandlerService', 'QuestionDataService',
+      '$scope', '$timeout', 'SolutionHandlerService', 'QuestionDataService',
       'LANGUAGE_PYTHON',
       function(
-          $scope, SolutionHandlerService, QuestionDataService,
+          $scope, $timeout, SolutionHandlerService, QuestionDataService,
           LANGUAGE_PYTHON) {
         var language = LANGUAGE_PYTHON;
-        var question = QuestionDataService.getData();
-        var prompts = question.getPrompts();
-        var currentPromptIndex = 0;
+        var questionSetId = 'strings';
```

Comment:
questionSetId will not always be 'strings', no? If not, add a TODO to assign the appropriate value based on a reaction to some future/as yet undefined user interaction?